homework_11/library.py

```diff
@@ -0,0 +1,130 @@
+"""Library"""
+
+
+class Book:
+    """Класс книга"""
+    def __init__(
+        self,
+        name: str,
```

Comment:
You are still using different approaches: if you want to use typehints, so please use it everywhere in the program.

Reply:
Oh, sorry, I didn't notice